scripts/charmm_to_fah.py

```diff
@@ -0,0 +1,309 @@
+"""
+Set up AurKA mutant simulations
+@author John D. Chodera
+@date 8 Aug 2014
+"""
+
+#
+# IMPORTS
+#
+from __future__ import division, print_function
+
+import os, os.path
+import copy
+import numpy
+import shutil
+import tempfile
+import numpy as np
+
+import pdbfixer
+from simtk import openmm, unit
+from simtk.openmm import app
+import mdtraj as md
+
+import sys
+
+# ParmEd Imports
+#from parmed.charmm import CharmmPsfFile, CharmmCrdFile, CharmmParameterSet
+from simtk.openmm.app import CharmmPsfFile, CharmmCrdFile, CharmmParameterSet
+
+from parmed import unit as u ### ???
+
+#
+
+
+print("Input PDB structure: 1OL5")
+pdbid = '1OL5'
+runs = 0
+
+output_path = '1OL5-phos-tpx2'
+
+# Source PDB
+pdbfilename = 'step2_solvator.pdb'
+
+print("Source PDB filename: %s" % pdbfilename)
+print("Output directory for mutations: %s" % output_path)
+
+#
+# PARAMETERS
+#
+
+universal_parameter_files = [
+    'toppar/par_all36_prot.prm',
+    'toppar/par_all36_na.prm',
+    'toppar/top_all36_prot.rtf',
+    'toppar/toppar_all36_na_nad_ppi.str',
+    'toppar/toppar_water_ions.str',
+    'step2_solvator.str',
+    'step2.2_ions.prm',
+    'step2.1_waterbox.prm',
+    'step1_pdbreader.str',
+    'step1_labelrot.str',
+    'toppar/toppar_all36_label_spin.str',
+    'step1_pdbreader.str',
+    'toppar/top_all36_cgenff.rtf',
+    'toppar/top_all36_na.rtf',
+    'toppar/toppar_all36_na_rna_modified.str',
+    'toppar/toppar_all36_prot_na_combined.str',
+    'toppar/par_all36_cgenff.prm',
+    'toppar/par_all36_na.prm',
+]
+
+psf_file = 'step2_solvator.psf'
+
+padding = 11.0 * unit.angstroms
+nonbonded_cutoff = 9.0 * unit.angstroms
+nonbonded_method = app.PME
+max_minimization_iterations = 10000
+temperature = 300.0 * unit.kelvin
+pressure = 1.0 * unit.atmospheres
+collision_rate = 5.0 / unit.picoseconds
+barostat_frequency = 50
+timestep = 2.0 * unit.femtoseconds
```

Comment:
I'd use a much smaller timestep of 1 or even 0.1 femtoseconds for an initial round of dynamics